lib/td/command/connector.rb

```diff
@@ -104,17 +104,23 @@ def connector_preview(op)
     $stdout.puts cmd_render_table(rows, :fields => fields, :render_format => op.render_format, :resize => false)
 
     $stdout.puts "Update #{config_file} and use '#{$prog} " + Config.cl_options_string + "connector:preview #{config_file}' to preview again."
-    $stdout.puts "Use '#{$prog} " + Config.cl_options_string + "connector:issue #{config_file}' to run Server-side bulk load."
+    $stdout.puts "Use '#{$prog} " + Config.cl_options_string + "connector:issue database_name table_name #{config_file}' to run Server-side bulk load."
   end
 
   def connector_issue(op)
-    database = table = nil
-    time_column      = nil
-    wait = exclude   = false
-    auto_create      = false
-
-    op.on('--database DB_NAME', "destination database") { |s| database = s }
-    op.on('--table TABLE_NAME', "destination table") { |s| table = s }
+    option_database = option_table = nil
+    time_column     = nil
+    wait = exclude  = false
+    auto_create     = false
+
+    op.on('--database DB_NAME', "(obsoleted)") { |s|
+      $stderr.puts '--database is obsolete option'
```

Comment:
It's very good to show this warning :+1:  But it's even better to show alternative way to do the same thing.